src/azul/plugins/metadata/hca/transform.py

```diff
@@ -139,6 +138,13 @@ def get_aggregator(cls, entity_type):
         else:
             return SimpleAggregator()
 
+    @classmethod
+    def post_process_aggregate(cls, contents: MutableJSON) -> None:
+        cell_count = sum(cs['total_estimated_cells']
+                         for cs in contents['cell_suspensions']
+                         if cs['total_estimated_cells'])
```

Comment:
```suggestion
                         if cs['total_estimated_cells'] is not None)
```

I don't think the optimization of not adding up `0` is not worth the lack of clarity.